extensions/window/internal.m

```diff
@@ -390,6 +443,26 @@ static int window__close(lua_State* L) {
     return window_pressbutton(L, kAXCloseButtonAttribute);
 }
 
+/// hs.window:focusTab(index) -> bool
+/// Method
+/// Focuses the tab in the window's tab group at index, or the last tab if
+/// index is out of bounds. Returns true if a tab was pressed.
+///
+/// Parameters:
+///  * index - A number, true if the window should be set fullscreen, false if not
+///
+/// Returns:
+///  * The `hs.window` object
+static int window_focustab(lua_State* L) {
+    AXUIElementRef win = get_window_arg(L, 1);
+    double indexDouble = luaL_checknumber(L, 2);
```

Comment:
Is there a reason you're using a double here?  Since you immediately put it into a CFIndex and don't use indexDouble again, it's probably cleaner to use `luaL_checkinteger` and assign it directly to `tabIndex`.